content/handbook/security/customer-support-operations/zendesk/users/end-users.md

```
@@ -0,0 +1,338 @@
+
+これは発生することが非常にまれで、おそらく非常にカスタマイズされたプロセスを要します。フルスタックエンジニアに状況を引き継いで対応してもらってください。
+
+### システムエンドユーザー設定を変更する {#modifying-system-end-user-settings}
+
+{{% alert title="危険" color="danger" %}}
+
+- サポートポータルの使い勝手に大きな影響を与える可能性があるため、極めて慎重に実行してください。
+- これは以下の 2 つのシナリオでのみ実行する必要があります:
+  - [エンドユーザーを BAN する](#banning-an-end-user)
+  - 対応する依頼 Issue（Feature Request、Administrative、Bug など）が存在する
+- システムエンドユーザー設定を変更した場合は、必ずこのページの[現在のシステムエンドユーザー設定](#current-system-end-user-settings)を更新してください
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)